Data/Vector.hs

```diff
@@ -169,7 +169,12 @@ import           Data.Vector.Mutable  ( MVector(..) )
 import           Data.Primitive.Array
 import qualified Data.Vector.Fusion.Bundle as Bundle
 
-import Control.DeepSeq ( NFData, rnf )
+import Control.DeepSeq ( NFData(rnf)
+#if __GLASGOW_HASKELL__ >= 821
```

Comment:
Use `#if MIN_VERSION_deepseq(1,4,3)` here instead of `#if __GLASGOW_HASKELL__ >= 821`, since it's entirely possible to use an `NFData1`-equipped `deepseq` with older versions of GHC than 8.2.

Reply:
You got it!